client/app/views/templates/menu.jade

```diff
@@ -12,3 +12,10 @@ li.calendars
         img.spinner(src="img/spinner.svg")
 
 ul#menuitems
+
+#multiple-actions.hidden
+    hr
+    p
+        = t('multiple actions')
+    i.remove-cals.fa.fa-trash(title=t('delete'), role="button")
+    i.export-cals.fa.fa-external-link(title=t('export'), role="button")
```

Comment:
I suggested we use the `fa-download` icon, instead of the `fa-external-link`, just to be consistent with other apps.